Dockerfile

```diff
@@ -1,48 +1,26 @@
-##########################################################
-#                  /!\ WARNING /!\                       #
-# This is completely experimental. Use at your own risk. #
-#             Also, learn you some docker:               #
-#           http://docker.io/gettingstarted              #
-##########################################################
```

Comment:
There was some existing docker stuff in this repo designed for self-hosters.  It should still be possible for self-hosters to use the modified docker file, but I'm going to fix any bugs in a demand-driven fashion rather than worry about maintaining strict compatibility here.